iamproperty/iamproperty.github.io
# New checkbox style

## Summary of Changes:
1. Modify the input vue component
2. Extra CSS to support tags
3. Document page for tags

## Rationale
Tag based multi-select input field.

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /elements/form/tags

## Ticket(s)
DS-72

## Checklist

## Notes
